launcher/ui/dialogs/ProgressDialog.cpp

```diff
@@ -54,10 +97,17 @@ ProgressDialog::~ProgressDialog()
 }
 
 void ProgressDialog::updateSize()
-{
+{   
+    QSize lastSize = this->size();
     QSize qSize = QSize(480, minimumSizeHint().height());
     resize(qSize);
     setFixedSize(qSize);
+    // keep the dialog in the center after a resize
+    if (lastSize != qSize)
+        this->move(
+            this->parentWidget()->x() + (this->parentWidget()->width() - this->width()) / 2,
+            this->parentWidget()->y() + (this->parentWidget()->height() - this->height()) / 2
+        );
 }
```

Comment:
I don't really think we should be doing this at all? Like, the window should just have some minimum sizes, depending on what is shown. But we probably shouldn't prevent the user from resizing this window, especially if there are multiple tasks visible in the dialog.

Reply:
You would think that that would work and itd hoped it would but the window layout flat out breaks after unhideing the scroll area unless we do a manual resize.

Reply:
NVM I fix it, the user can now resize the dialog the function now sets the minimum size and recenters only after unhideing